packages/drivers/routerlicious-driver/src/test/r11sSocketTests.spec.ts

```diff
@@ -48,6 +49,35 @@ describe("R11s Socket Tests", () => {
 		scopes: [],
 	};
 
+	// Defines error scenarios in a structured way to avoid test code repetition.
+	const errorScenarios = [
+		{
+			name: "Token Revoked",
```

Comment:
Nit: it would probably be a good idea to define an interface for this. Adds some boilerplate, but it also gives a place where you can document what these various properties are for future maintainers.